src/designProject/Originator.java

```diff
@@ -1,3 +1,5 @@
+package designProject;
+
 import java.util.ArrayList;
 
 /**
```

Comment:
Types of attributes should use interfaces from Collections framework.  (i.e. List instead of ArrayList for the type).

This isn't the Memento pattern.  For the Memento pattern should have a public static nested class.  That class should be public, but every method, constructor, and attribute should be private.